client/src/Votes.js

```diff
@@ -0,0 +1,26 @@
+import React, {useState} from 'react';
+import Dislike from "./Dislike"
+import Like from "./Like"
+
+const Votes = () => {
+    const [likes, setLikes] = useState(0)
+
+	function liked(){
+		setLikes(likes + 1)
+	}
+
+	function Disliked(){
```

Comment:
Another function starting with a capital here.

Reply:
Changed. Thankyou for taking the time to review my code. It is good to get feedback so that I can improve.